velox/exec/tests/AggregationTest.cpp

```diff
@@ -1442,6 +1456,20 @@ TEST_F(AggregationTest, groupingSetsEmptyInput) {
           .planNode();
 
   assertQuery(plan, makeRowVector({makeFlatVector<int64_t>({0, 0})}));
+
+  // No aggregations in the GROUPING SET.
```

Comment:
Is this the same test as above but using single agg vs. partial + final?

Reply:
No. It also has 2 global grouping sets. The above test had only one.